### ISSUE TITLE
Duplicate enum class documentation

### ISSUE THREAD

**Issue by sschuberth** [pwall567/json-kotlin-schema-codegen#39]
Converting https://www.aosd.cloud.audi/jsonschemadoc/static/aosd.schema.json using the Kotlin script

```
#!/usr/bin/env kotlin

@file:DependsOn("net.pwall.json:json-kotlin-schema-codegen:0.106")

import java.io.File
import net.pwall.json.schema.codegen.CodeGenerator

val codeGenerator = CodeGenerator()
codeGenerator.baseDirectoryName = "output/directory"
codeGenerator.basePackageName = "com.example"
codeGenerator.generate(File("/home/sebastian/Downloads/aosd.schema.json"))
```

gives 

```
    /**
     * Identifying the origin of the license information.
     */
    /**
     * Identifying the origin of the license information.
     */
    enum class Origin {
        packagemanagement,
        scm,
        licensefile
    }
```

so the enum class documentation is duplicated.

**Comment by pwall567** [comment]
Hi @sschuberth , thank you for bringing this to my attention.

I've been working on enums lately, making nested enums share a template with standalone enums, and obviously I failed to notice this.

I have fixed this in version 0.107, and I am closing this issue.  If you continue to have problems feel free to reopen this issue or raise a new one.

**Comment by sschuberth** [comment]
Thanks for the quick fix @pwall567! I can confirm version 0.107 to fix the issue.
